packages/node-runtime-worker-thread/src/worker-runtime.spec.ts

```diff
@@ -20,7 +20,10 @@ const workerThreadModule = fs.readFile(
   'utf8'
 );
 
-describe('worker', () => {
+// This set of tests causes flakiness in CI, disabled for now and will be
+// resolved in a separate PR
+// eslint-disable-next-line mocha/no-skipped-tests
```

Comment:
I think eslint should only print a warning, so it should be okay to let it do that (and might actually be a good thing because it keeps reminding us that something's still wrong)

Reply:
Ahh, good point, sorry, I thought it was an error, not a warning, will remove it